Source/Dafny/DafnyAst.cs

```diff
@@ -3416,14 +3417,7 @@ public class ModuleSignature {
 
     // Final projection is for module export
     public bool FindExport(string name, out ModuleExportDecl pp) {
-      TopLevelDecl top;
-      if (TopLevels.TryGetValue(name, out top) && top is ModuleExportDecl) {
-        pp = ((ModuleExportDecl)top);
-        return true;
-      } else {
-        pp = null;
-        return false;
-      }
+      return ExportSets.TryGetValue(name, out pp);
```

Comment:
Can we just delete this method?  If you're exporting `ExportSets` as `public` anyway, this seems unnecessary.

Reply:
Good idea.